src/middleware/environment.js

```diff
@@ -117,13 +115,18 @@ module.exports = function environmentMiddleware(argv) {
       module: Object.assign({}, module, { appName: env.appName }),
       network: configureNetwork(argv, env.network),
     }
+
     if (env.registry) {
       resp.apmEnsRegistry = env.registry
       if (apm) {
         apm['ens-registry'] = env.registry
       }
     }
 
+    if (env.wsRPC) {
```

Comment:
I wanted to say this is a breaking change, since now you need the `wsRPC` property to do stuff like:
`dao apps 0xF9e4c6ee977b1Fe7D296FbE3b75BA05f12A30E4e --environment staging`
But this doesn't seem to work on master anyway, it gets stuck at _Inspecting DAO_.

1. Did dao commands ever work on something else than localhost ?
2. Is the wsRPC really necessary? I don't understand why the wrapper cannot fetch data using an http provider :confused:

It would have been nice to not need this extra `wsRPC` property, since I assume the cli doesn't need sandboxing like the dapp does.

Reply:
@0x6431346e Answering 2: To use the wallet-provider Jeremy builded we need to have `wsRPC` arg to pass the custom provider.

Here is the wallet-provider [Readme](https://github.com/jvluso/aragon.js/tree/wallet-provider-2/packages/aragon-wallet-provider)

Reply:
In web3.js 1.0 only ws providers can listen to events unfortunately. So yeah, this has never worked unless using frame that has a ws provider